src/commands.ts

```diff
@@ -51,3 +51,51 @@ export const resetCommand = async () => {
 	info("Biome extension was reset");
 	await start();
 };
+
+export const initializeWorkspaceCommand = async (args: Uri) => {
+	const subconfigs = [
+		"[javascript]",
+		"[typescript]",
+		"[typescriptreact]",
+		"[javascriptreact]",
+		"[json]",
+		"[jsonc]",
+		"[vue]",
+		"[astro]",
+		"[svelte]",
+		"[css]",
+		"[graphql]",
+	].join("");
+
+	// Scopes the config down to the current workspace folder
+	const config = workspace.getConfiguration(undefined, Uri.parse(args.path));
+	try {
+		const configsToUpdate = [
+			{ name: "biome.enabled", value: true },
+			{
+				name: "editor.codeActionsOnSave",
+				value: {
+					...config.get<object | undefined>(
+						"editor.codeActionsOnSave",
+					),
+					"source.organizeImports.biome": "always",
+					"quickfix.biome": "always",
+				},
+			},
+			{ name: "editor.defaultFormatter", value: "biomejs.biome" },
+			{
+				name: `${subconfigs}`,
```

Comment:
Jake Bailey implied that this trick may only work for certain languages. Might want to double check that this works for the new ones you've added (graphql, svelte, etc)
 https://github.com/microsoft/TypeScript/blob/ca18009b8bd5cea6a7fbcba3d7e4dd3c4633bc92/.vscode/settings.template.json#L10